package.json

```
@@ -5,6 +5,7 @@
     "packages/*"
   ],
   "scripts": {
+    "dev": "make dev",
```

Any reason why we wouldn't directly stop using Makefile and set up whatever `make dev` does here?
 i.e.:
```suggestion
    "dev": "lerna run dev --stream --parallel",
```

---

To be honest I didn't even look at what make dev was doing, just quickly added the script entry.

---

i think this makes sense